dcpy/connectors/edm/publishing.py

```diff
@@ -194,18 +194,19 @@ def publish(
     acl: s3.ACL,
     publishing_version: str | None = None,
     keep_draft: bool = True,
-    target_bucket: str | None = None,
```

Comment:
This seemed unnecessary, and easy enough to add back in if we ever move publishing.

Or actually, should we keep for kpdb?

Reply:
down to drop it. I just checked and KPDB doesn't use publishing.py since we never publish it. it's currently using `s3.upload_folder()`

(future thoughts) seems like we might wanna pick between file-level access via ACL and bucket-level access via multiple similar buckets

Reply:
Oh yeah this would be relevant if kpdb drafts were going to `edm-publishing` and then we were publishing to private, but that obviously makes no sense :)